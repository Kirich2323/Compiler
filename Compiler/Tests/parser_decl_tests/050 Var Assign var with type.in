type
    b = integer;
var
    a : integer = b;