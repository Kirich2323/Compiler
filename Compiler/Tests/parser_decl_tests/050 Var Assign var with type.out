(4 ; 20): Invalid expression.